app/src/main/java/com/vero/tasky/agenda/data/local/AgendaDatabase.kt

```diff
@@ -2,21 +2,21 @@ package com.vero.tasky.agenda.data.local
 
 import androidx.room.Database
 import androidx.room.RoomDatabase
-import androidx.room.TypeConverters
-import com.vero.tasky.agenda.data.local.converter.Converters
-import com.vero.tasky.agenda.data.local.dao.DeletedAgendaItemDao
-import com.vero.tasky.agenda.data.local.dao.EventDao
-import com.vero.tasky.agenda.data.local.dao.ReminderDao
-import com.vero.tasky.agenda.data.local.dao.TaskDao
+import com.vero.tasky.agenda.data.local.dao.*
 import com.vero.tasky.agenda.data.local.entities.*
 
-@Database(entities = [EventEntity::class, ReminderEntity::class, TaskEntity::class,
-    LocalPhotoEntity::class, RemotePhotoEntity::class, DeletedAgendaItemEntity::class,
-    AttendeeEntity::class], version = 1)
-@TypeConverters(Converters::class)
+@Database(entities = [AttendeeEntity::class, CreatedAttendeeEntity::class, CreatedEventEntity::class,
+    CreatedLocalPhotoEntity::class, CreatedReminderEntity::class, CreatedTaskEntity::class,
+    DeletedEventEntity::class, DeletedLocalPhotoEntity::class, DeletedReminderEntity::class,
+    DeletedTaskEntity::class, EventEntity::class, ReminderEntity::class,
+    RemotePhotoEntity::class, TaskEntity::class, UpdatedAttendeeEntity::class,
+    UpdatedEventEntity::class, UpdatedLocalPhotoEntity::class, UpdatedReminderEntity::class,
+    UpdatedTaskEntity::class], version = 1)
```

Comment:
This approach of having one separate table for each type of modification for each type of agenda item doesn't scale well. Adding one additional type of agenda item means you have to add at least 4 tables to your DB. What about just one table?
```
@Entity
data class ModifiedAgendaItem(
    @PrimaryKey(autoGenerate = false) val id: String,
    val agendaItemType: AgendaItemType, // can be EVENT, TASK or REMINDER
    val modificationType: ModificationType // can be CREATED, DELETED or UPDATED
)
```
If you now add an additional AgendaItem type, all you need to do is add an extra enum entry to `AgendaItemType` and that's it. Every change in a table will require a DB migration, so you better want to avoid this if you can.